CMakeLists.txt

```diff
@@ -85,9 +85,6 @@ function(openvino_developer_export_targets)
         "A list of OpenVINO exported components" FORCE)
 endfunction()
 
-ie_cpack_add_component(ngraph REQUIRED)
-ie_cpack_add_component(ngraph_dev REQUIRED DEPENDS ngraph)
```

Comment:
if it removes cpack generation for this compoent please revert this lines. This cpack is used for conda packages.

Reply:
they are just moved close to ngraph (see `src/core/CMakeLists.txt`)